login-form/src/components/page-components/Dashboard.js

```diff
@@ -62,11 +68,11 @@ const Dashboard = ({
 const mapStateToProps = (state) => {
   console.log("state", state);
   return {
```

Comment:
```suggestion
const mapStateToProps = (state) => ({
});
```

you can simplify this by returning the object directly.

Reply:
this comment is not addressed right @sasiyaxv ?